app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package com.worldsnas.starplayer.view.musics_list
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.worldsnas.starplayer.core.BaseViewHolder
+import com.worldsnas.starplayer.databinding.ItemMusicBinding
+import com.worldsnas.starplayer.model.Music
+
+class MusicsListAdapter(private val context: Context, private val musics: ArrayList<Music>) :
+    RecyclerView.Adapter<BaseViewHolder<Music>>() {
+
+    class MusicListItemViewHolder(private val mBinding: ItemMusicBinding) :
```

Comment:
man basham baraye rahat khondan in class ro biron e adapter misazam 
masalan payine hamin file